### ISSUE TITLE
yadcf column data fill with ajax source

### ISSUE THREAD

**Issue by cygnustr** [vedmack/yadcf#412]
Question should be asked on stackoverflow and tagged with yadcf - http://stackoverflow.com/questions/tagged/yadcf , that way more people
will be able to help you and your questions will be easier to find for others

Use github for bug reports or new features request only

In case of bug always post a code sample that resembles your issue or event better provide a link to your page

**Comment by cygnustr** [comment]
Hello,

how to fill yadcf column data with ajax source. 



`

data:[
                    $.ajax({
                        url: "../ajax.php?islem=getleaddurumu",
                        type: "GET",
                        success: function (results) {
                            $.each(results, function( label, value ) {
                                resss.push({value:this.value,label:this.label});
                            });
                            console.log(resss);
                           // return resss;
                        },
                        dataType: "json",
                    })
`

**Comment by vedmack** [comment]
post your question on stackoverflow, please use github for reporting bugs or code contributing
